server/src/main/java/org/candlepin/resource/OwnerContentResource.java

```diff
@@ -289,6 +298,12 @@ public ContentData updateContent(@PathParam("owner_key") String ownerKey,
         }
 
         existing = this.contentManager.updateContent(content, owner, true);
+        try {
+            ownerManager.refreshOwnerForContentAccess(owner);
```

Comment:
It only removes the cached content block in this call. The number of consumers does not change the timing for new content.

Working on the deletion of CACerts on the consumers via db call.